app/js/actions/careSetting.js

```diff
@@ -1,51 +1,43 @@
 import axios from 'axios';
 import {
-  INPATIENT_CARESETTING,
-  INPATIENT_CARESETTING_ERROR,
-  OUTPATIENT_CARESETTING,
-  OUTPATIENT_CARESETTING_ERROR,
+  PATIENT_CARESETTING,
```

Comment:
Could you please make this more descriptive? Say `PATIENT_CARESETTING_SUCCESS`

Reply:
That makes sense.
Would notify the team about this suggestions and make changes based on the team's decision.

Reply:
Have implemented new changes based on feedback from the team. The action type is now `PATIENT_CARESETTING_SUCCESS`.